lib/src/blaze/sync_status.rs

```diff
@@ -53,36 +52,18 @@ impl SyncStatus {
     pub fn finish(&mut self) {
         self.in_progress = false;
     }
-
-    fn perct(&self, num: u64) -> u8 {
-        let a = if self.blocks_total > 0 {
-            let (b, d) = if self.batch_total > 0 {
-                ((self.batch_num * 100 / self.batch_total), self.batch_total)
-            } else {
-                (0, 1)
-            };
-            let p = cmp::min(((num * 100) / self.blocks_total) as u8, 100);
-            b + (p as usize / d)
-        } else {
-            0
-        };
-
-        cmp::min(100, a as u8)
-    }
 }
 
 impl fmt::Display for SyncStatus {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         if self.blocks_total > 0 && self.in_progress {
             write!(
                 f,
-                "id: {}, blocks: {}% ({}), decryptions: {}% ({}), tx_scan: {}% ({}), anchors: {}",
+                "id: {}, blocks_done/blocks_total: {:4}/{:4}, decryptions: {:4}, tx_scan: {:4}, anchors: {}",
```

Comment:
This display used to show the number of blocks done in the batch, as well as the total percentage of blocks done. (same with decryptions). Now, with the removal of `perct`, we see the number of blocks done in the batch, and the number of blocks in the batch, but we no longer see any obvious progress from batch to batch.

Reply:
This is incoherent.   "Total percentage of _WHICH_ blocks done?"

All blocks on the chain?
Blocks anticipated for this particular sync?
Blocks in this batch?

Is it's intended to that the number of blocks scheduled for this sync should be displayed then I will make that a mut field of sync_status (if it's not already) and report that number (which increases on batch completion).

If someone wants to subsequently throw out the denominator to report a less informative % later in life, they can do so, since all of the relevant information will be available and clearly labeled.

Notice that they could _NOT_ go in the opposite direction if I just reported the %.

Reply:
`sync_id` `blocks_done` `trial_dec_done` `blocks_total` `txn_scan_done` `orchard_anchors_done`


Do we need to add more?